File: Codeforces/SET7/E.py
a, b, c = map(int, input().split())
arr = [0 for _ in range(a + b + c + 1)]
for i in range(1, a + 1):
    for j in range(1, b + 1):
        for t in range(1, c + 1):
            arr[i + j + t] += 1
maxx, pos = -1e99, 0
for i in range(1, a + b + c + 1):
    if maxx < arr[i]:
        maxx = arr[i]
        pos = i 
print(pos)

File: Codeforces/SET7/E.java
import java.util.*;

public class E {
    public static void main (String[] args){
        Scanner s = new Scanner(System.in);
        int x, y, z;
        x = s.nextInt();
        y = s.nextInt();
        z = s.nextInt();
        int [] a = new int[x + y + z + 1];
        for (int i = 1; i <= x; i += 1)
            for (int j = 1; j <= y; j += 1)
                for (int t = 1; t <= z; t += 1)
                    a[i + j + t]+=1;
        int maxx = -1, pos = 0;
        for (int i = 1; i < x + y + z + 1; i += 1){
            if (maxx < a[i]){ 
                maxx = a[i];
                pos = i;
            }
        }
        System.out.println(pos);
    }
}